packages/sdk-bundles-api/src/replay/bidirectional/README.md

```diff
@@ -0,0 +1,14 @@
+### APIs used for messages sent from the replay and record bundles to the SDK
+
+This folder contains APIs used both for messages sent from the Metic SDK to the bundle, and for messages sent from the bundle to Metic SDK.
```

Comment:
"Metic"?

Reply:
This was copied and pasted. I'll rename it across them all.